ci_jobs_trigger/libs/openshift_ci/ztream_trigger/zstream_trigger.py

```diff
@@ -55,7 +55,7 @@ def already_processed_version(
     return False
 
 
-def trigger_jobs(config: Dict, jobs: List, logger: logging.Logger) -> bool:
+def trigger_jobs(config: Dict, jobs: List, zstream_version: str, logger: logging.Logger) -> bool:
```

Comment:
When you add a new arg, add it last. if someone is using posstional args, this will break their code.

Reply:
will keep this in mind, thank you